torq.sh

```diff
@@ -67,13 +67,19 @@ startline() {
  }
 
 start() {
-  if [[ -z $(findproc "$1") ]]; then                                                                # check process not running
-    sline=$(startline "$1")                                                                         # line to run each process
-    echo "$(date '+%H:%M:%S') | Starting $1..."
-    eval "nohup $sline </dev/null >${KDBLOG}/torq${1}.txt 2>&1 &"                                   # run in background and redirect output to log file
+  procno=$(awk '/,'$1',/{print NR}' "$CSVPATH")
+  host=$(getfield "$procno" "host")
+  if [[ $host == "localhost" || $host == $HOSTNAME ]]; then
```

Comment:
This is great, all seems to work as expected! Small caveat I realised from the ports pr is that `torq.q` allows also IP address as a valid option so that may need to be included here! Can be done with `hostname -i` but that may not work on some systems.